lxd/storage/drivers/driver_alletra_wsapi1.go

```diff
@@ -274,3 +274,43 @@ func (p *alletraClient) login() error {
 	p.sessionKey = respBody.Key
 	return nil
 }
+
+func (p *alletraClient) createVolumeSet() error {
+	req := map[string]any{
+		"name":    p.driver.name,
+		"comment": "Created and managed by LXD",
```

Comment:
Just leaving a note here to ensure we consider and implement adding a `comment` for volumes and snapshots too - what will be in that comment still remains to be discussed.